app/containers/message/Attachments.tsx

```diff
@@ -50,7 +48,7 @@ const AttachedActions = ({ attachment }: { attachment: IAttachment }) => {
 	});
 	return (
 		<>
-			<Text style={[styles.text, { color: themes[theme].bodyText }]}>{attachment.text}</Text>
+			<Markdown msg={attachment.text} getCustomEmoji={getCustomEmoji} theme={theme} />
```

Comment:
```suggestion
			<Markdown msg={attachment.text} getCustomEmoji={getCustomEmoji} />
```
You don't need to pass the theme as the component is wrapped by `withTheme`